clients/chain/chain.go

```diff
@@ -0,0 +1,77 @@
+package chain
+
+import (
+	"fmt"
+	"strings"
+)
+
+const (
+	ethToElrondNameTemplate         = "%sToElrond"
+	elrondToEthNameTemplate         = "ElrondTo%s"
+	baseLogIdTemplate               = "%sElrond-Base"
+	elrondClientLogIdTemplate       = "%sElrond-ElrondClient"
+	elrondDataGetterLogIdTemplate   = "%sElrond-ElrondDataGetter"
+	ethClientLogIdTemplate          = "%sElrond-%sClient"
+	elrondRoleProviderLogIdTemplate = "%sElrond-ElrondRoleProvider"
+	ethRoleProviderLogIdTemplate    = "%sElrond-%sRoleProvider"
+	broadcasterLogIdTemplate        = "%sElrond-Broadcaster"
+)
+
+// Chain defines all the chain supported
+type Chain string
+
+const (
+	Elrond   Chain = "Elrond"
+	Ethereum Chain = "Ethereum"
+	Bsc      Chain = "Bsc"
+)
+
+// ToLower returns the lowercase string of chain
+func (c Chain) ToLower() string {
+	return strings.ToLower(string(c))
+}
+
+// EthToElrondName return the string using chain value and ethToElrondNameTemplate
```

Comment:
```suggestion
// EthToElrondName returns the string using chain value and ethToElrondNameTemplate
```
valid on all methods

Reply:
done